src/test/determinism/determinism1.test.shadow.config.yaml

```diff
@@ -1,32 +1,28 @@
-options:
-  stoptime: 5
+general:
+  stop_time: 5
 topology:
-- graphml: |
+  graphml: |
     <graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd">
-      <key attr.name="packetloss" attr.type="double" for="edge" id="d4" />
+      <key attr.name="packet_loss" attr.type="double" for="edge" id="d4" />
       <key attr.name="latency" attr.type="double" for="edge" id="d3" />
-      <key attr.name="bandwidthup" attr.type="int" for="node" id="d2" />
-      <key attr.name="bandwidthdown" attr.type="int" for="node" id="d1" />
-      <key attr.name="countrycode" attr.type="string" for="node" id="d0" />
+      <key attr.name="bandwidth_up" attr.type="string" for="node" id="d2" />
+      <key attr.name="bandwidth_down" attr.type="string" for="node" id="d1" />
+      <key attr.name="country_code" attr.type="string" for="node" id="d0" />
       <graph edgedefault="undirected">
         <node id="poi-1">
           <data key="d0">US</data>
-          <data key="d1">10240</data>
-          <data key="d2">10240</data>
+          <data key="d1">81920 Kibit</data>
+          <data key="d2">81920 Kibit</data>
```

Comment:
I think you want this to be `Kibibit` instead of `Kibit`?
https://en.wikipedia.org/wiki/Kibibit

This same problem applies to all of the yaml files in the PR that contain topology graphml code. I'm only noting it here, but you should change it everywhere.

Reply:
Do you want to change this to `Kibibit` just as a preference, or is there an issue with `Kibit`? Right now the configuration parsing will accept units in both `kibibit` (the full unit name) and `Kibit` (the unit symbol).

Reply:
Oh shit, this is my bad, I had never seen `Kibit` before and I missed this part on the wiki page:

> The unit symbol of the kibibit is Kibit

So never mind, I think how you have it is fine.

Reply:
> So never mind, I think how you have it is fine.

Okay, and if you would still prefer "kibibit" just let me know, it's an easy change.